.github/workflows/test.yml

```diff
@@ -19,9 +19,7 @@ jobs:
           cache: 'npm'
       - run: npm ci
       - run: npm run build
-      # v4 download seems to have some flakiness with the download of artifacts so pinning to v3 for now
-      # @see https://github.com/actions/download-artifact/issues/249
```

Comment:
https://github.com/actions/download-artifact/issues/249#issuecomment-2040750798 says
> Since the fix was released ~3 months ago I haven't faced the timeout issue again, using v4. Also, the lack of activity in here implies that it was indeed fixed across the board IMHO